public/wp-content/themes/glific/page-contact.php

```diff
@@ -38,8 +38,8 @@
 						<?php echo $get_help_section['development_and_design_title']; ?>
 					</h5>
 					<div class="position-absolute bottom-8 d-flex flex-column bottom-1">
-						<a class="glific-button-border bg-theme-primary py-4 px-2 w-190 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none mt-5 mb-5" target="_blank" href="<?php echo $get_help_section['discord_link']; ?>"><?php echo $get_help_section['discord_title']; ?></a>
-						<a class="glific-button-border bg-theme-secondary py-4 px-2 w-190 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none px-6" target="_blank" href="<?php echo $get_help_section['book_a_demo_link']; ?>"><?php echo $get_help_section['book_a_demo_title']; ?></a>
+						<a class="glific-button-border bg-theme-primary py-4.5 px-9 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none mt-5 mb-5" target="_blank" href="<?php echo $get_help_section['discord_link']; ?>"><?php echo $get_help_section['discord_title']; ?></a>
+						<a class="glific-button-border bg-theme-secondary py-4.5 px-9 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none" target="_blank" href="<?php echo $get_help_section['book_a_demo_link']; ?>"><?php echo $get_help_section['book_a_demo_title']; ?></a>
```

Comment:
@wevekamola it would be good to reduce the line length for more readability. Something like this may help.

```suggestion
						<a class="glific-button-border bg-theme-primary py-4.5 px-9 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none mt-5 mb-5"
						    target="_blank"
						    href="<?php echo $get_help_section['discord_link']; ?>">
						        <?php echo $get_help_section['discord_title']; ?>
						</a>
						<a class="glific-button-border bg-theme-secondary py-4.5 px-9 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none"
						    target="_blank"
						    href="<?php echo $get_help_section['book_a_demo_link']; ?>">
						        <?php echo $get_help_section['book_a_demo_title']; ?>
						</a>
```